tasks/copy_to_glacier/handler.py

```diff
@@ -79,14 +80,28 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
                     bucket (str)
```

Comment:
The docstring for this function is no longer accurate, but this is the earliest github allows me to comment.

Reply:
Description needs a tweak, and the environment variable should be mentioned.

Reply:
Updated doc string and added environment variable information.